setup.py

```diff
@@ -47,7 +47,7 @@
             'pyro-api>=0.1.1'
         ],
         'dev': [
-            'funsor @ git+https://github.com/pyro-ppl/funsor.git@6575ac2c3f7ac25a6a1e5f1107b1b7c072edd992',
+            'funsor',
```

Comment:
So the latest release of funsor has this commit?

Reply:
oh, good question! I need to verify this...

Reply:
yes, it has.